nixos/modules/misc/ids.nix

```diff
@@ -414,6 +415,7 @@
       btsync = 113;
       #minecraft = 114; # unused
       #monetdb = 115; # unused (not packaged), removed 2016-09-19
+      vault = 115;
```

Comment:
But is there a point to using a static id *and* adjusting the ownership? I can think of some, but that'd depend on the kind of data generated.